Octokit.Tests.Integration/Clients/AssigneesClientTests.cs

```diff
@@ -4,16 +4,19 @@
 using Octokit.Tests.Integration;
 using Octokit.Tests.Integration.Helpers;
 using Xunit;
+using System.Collections.Generic;
 
 public class AssigneesClientTests
 {
     readonly IGitHubClient _github;
     readonly RepositoryContext _context;
+    private readonly IIssuesClient _issuesClient;
 
     public AssigneesClientTests()
     {
         _github = Helper.GetAuthenticatedClient();
         var repoName = Helper.MakeNameWithTimestamp("public-repo");
+        _issuesClient = _github.Issue;
```

Comment:
If this is only referenced once, you might as well inline it into the method on line 51. Otherwise there's two usages in the test that can be replaced with this field. Up to you.